### ISSUE TITLE
hub launch throwing memory errors and is very slow

### ISSUE THREAD

**Issue by lwasser** [earthlab/hub-ops#341]
@kcranston i'm trying to open my pod and running into some issues. i get this error in the logs - i'm wondering if someone is trying to open something large on another pod?

<img width="1134" alt="Screen Shot 2021-04-14 at 2 52 52 PM" src="https://user-images.githubusercontent.com/7649194/114777797-503c0080-9d31-11eb-9927-b96121ba6d88.png">

**Comment by lwasser** [comment]
<img width="780" alt="Screen Shot 2021-04-14 at 2 55 30 PM" src="https://user-images.githubusercontent.com/7649194/114777932-795c9100-9d31-11eb-8e2c-9e58daa3ebf0.png">

**Comment by kcranston** [comment]
I was just thinking this afternoon that the hub had been performing so well... ha

Is this still happening? I just tried logging in and did not have a problem. I can check the logs / console for other errors.

**Comment by lwasser** [comment]
let me try again @kcranston i wasn't able to login yesterday.

**Comment by lwasser** [comment]
i noticed a student on the hub with nate and wondered if they tried to open some really big data while i was trying to login. it works now. let me ask nate if he was working with that student.

**Comment by lwasser** [comment]
i guess i will close this for  now,  and i'll come back to  it  and  reopen if it happens again.
